assets/scripts/streets/__tests__/StreetMetaWidthMenu.test.js

```diff
@@ -64,6 +64,7 @@ describe('StreetMetaWidthMenu', () => {
       />
     )
 
+    // cant change to user event becuase test relies on firing the change event
```

Comment:
@louh you might have a better idea about how to change this, but I tried and didn't have much luck. Might have to refactor the test itself, which may not be possible or worth the effort currently.

Reply:
You should be able to use [`selectOptions`](https://github.com/testing-library/user-event#selectoptionselement-values):

```js
    userEvent.selectOptions(screen.getByRole('combobox'), '40')
```

Reply:
Ah, gotcha!